src/app/core/toolbar/toolbar.component.ts

```diff
@@ -53,7 +53,7 @@ export class ToolbarComponent implements OnInit, OnDestroy, DoCheck  {
 
   ngOnInit() {
     this.subscriptions.push(this.projects$.subscribe((projects) => {
-      this.projects = projects;
+      this.projects = cloneDeep(projects);
```

Comment:
Could you please tell me the reason we need cloneDeep of projects here?

Reply:
i thing i forgot to remove that in the commit i made i will remove it immediately